### PR TITLE
Show timestamp inside bubble for text messages

### LINKED ISSUE #3927: Update timestamps according to latest designs
### Description

*Type*: Bug

*Summary*: Update timestamps due to newest mocks 

#### Expected behavior

image

#### Actual behavior
timestamps are out of the message bubbles

### Additional Information
* Status version: develop
* Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by cammellos**
fixes #3927 

### Summary:

Shows timestamp inside the bubble for text messages, updates also the padding according to designs, remove avatar from 1-to-1 chat and moves it to toolbar.
status: ready

### CODE REVIEW

File: src/status_im/chat/styles/message/message.cljs [hunk 1]
```diff
@@ -48,13 +48,18 @@
...
-(def message-timestamp
-  {:margin-left     5
-   :margin-right    5
-   :margin-bottom   -2
-   :color           colors/gray
-   :opacity         0.5
-   :align-self      :flex-end})
+(defn message-timestamp [justify-timestamp?]
+  (cond-> {:color      colors/gray
+           :font-size  10
```
Comment (jeluard) [inline src/status_im/chat/styles/message/message.cljs]: I find using plain merge and when more readable for those cases.

File: src/status_im/chat/styles/message/message.cljs [hunk 2]
```diff
@@ -117,7 +122,10 @@
 
 (defn message-view
   [{:keys [content-type outgoing group-chat selected]}]
-  (merge {:padding         12
+  (merge {:padding-top     6
+          :padding-right   12
+          :padding-bottom  8
+          :padding-left    12
```
Comment (jeluard) [inline src/status_im/chat/styles/message/message.cljs]: You can use `padding-horizontal` here.

File: src/status_im/chat/views/message/message.cljs [hunk 3]
```diff
@@ -160,18 +164,19 @@
                                           (autolink string event-on-press)))
                                  text-seq))))
 
+(defn timestamp-with-padding [t]
+  (str "   " t))
```
Comment (jeluard) [inline src/status_im/chat/views/message/message.cljs]: Could you add a comment detailing why this is not done in CSS?

**Review by jeluard — CHANGES_REQUESTED** [review]

**Review by rasom — APPROVED** [review]

**Comment by asemiankevich** [comment]
@cammellos does the PR fixes the following:

1. timestamps in bubbles
2. remove avatars
3. correct paddings
4. change messages colors? 

I am sorry if this was not readable in the issue description, me bad :(  Would you mind to check this PR and adjust?

**Comment by cammellos** [comment]
not all of them, but i can add the missing parts

**Review by janherich — APPROVED** [review]

**Comment by asemiankevich** [comment]
@cammellos thanks!!

**Comment by denis-sharypin** [comment]
@cammellos Please don't change the color of the sender bubble. Sorry, mockups are misleading in this way. cc @asemiankevich

**Comment by cammellos** [comment]
@jeluard made the changes, could you have a quick look? thanks

File: src/status_im/chat/views/toolbar_content.cljs [hunk 4]
```diff
@@ -64,11 +65,14 @@
...
-            contact                               [:get-current-chat-contact] 
+            contact                               [:get-current-chat-contact]
             sync-state                            [:sync-state]]
-    [react/view common.styles/flex
+    [react/view {:style st/toolbar-container}
+
+     [react/view (when-not group-chat [photos/member-photo public-key])]
      [react/view (st/chat-name-view (or (empty? accounts)
                                         show-actions?))
+
```
Comment (jeluard) [inline src/status_im/chat/views/toolbar_content.cljs]: Extra lines?

**Review by jeluard — APPROVED** [review]

**Comment by jeluard** [comment]
@cammellos Thanks!

**Comment by asemiankevich** [comment]
@cammellos the avatar on the top is not being changed (in chat). However new one is shown on Home tab

image

image

Also names of the participants are outside the bubbles but @denis-sharypin said it is okay for now

**Comment by cammellos** [comment]
@asemiankevich how did you trigger a change in the picture? Just changing the picture on phone B will do?

**Comment by asemiankevich** [comment]
@cammellos so i created 1x1 chat with default pictures under Ropsten and then i went to devices' profiles and changed default pictures to custom

**Comment by cammellos** [comment]
@asemiankevich fixed here https://jenkins.status.im/blue/organizations/jenkins/status-react%2Fpull%20requests/detail/PR-3970/13/pipeline 
thanks!

<blockquote></blockquote>

**Comment by asemiankevich** [comment]
it works now @cammellos thanks!

**Pull request merged**